rootfs/api/models/app.py

```diff
@@ -800,11 +804,11 @@ def _build_env_vars(self, release):
             default_env['BUILDER_STORAGE'] = settings.APP_STORAGE
             default_env['DEIS_MINIO_SERVICE_HOST'] = settings.MINIO_HOST
             default_env['DEIS_MINIO_SERVICE_PORT'] = settings.MINIO_PORT
+            default_env['SOURCE_VERSION'] = release.build.sha
```

Comment:
should we check for sha and add as environment as it is available even for dockerfile apps